crates/web-sys/src/features/gen_ReadableStream.rs

```diff
@@ -73,6 +73,28 @@ extern "C" {
         transform: &ReadableWritablePair,
         options: &StreamPipeOptions,
     ) -> ReadableStream;
+    #[cfg(web_sys_unstable_apis)]
```

Comment:
I'm not certain this is correct. `pipe_to` takes a `WritableStream` (unstable) as a parameter.